packages/strapi-plugin-upload/admin/src/translations/zh.json

```diff
@@ -0,0 +1,82 @@
+{
+  "button.next": "繼續",
+  "checkControl.crop-duplicate": "複製及裁切素材",
+  "checkControl.crop-original": "裁切原始素材",
+  "control-card.add": "增加",
+  "control-card.cancel": "取消",
+  "control-card.copy-link": "複製連結",
+  "control-card.crop": "裁切",
+  "control-card.delete": "刪除",
+  "control-card.download": "下載",
+  "control-card.edit": "編輯",
+  "control-card.replace-media": "取代原本的媒體檔",
+  "control-card.save": "儲存",
+  "filter.add": "增加過濾條件",
+  "filter.name": "過濾條件",
```

Comment:
Same here, instead you need to add the translation in the `strapi-admin` package as `"app.utils.filters"`.